oak_functions/sdk/oak_functions/Cargo.toml

```diff
@@ -5,23 +5,21 @@ authors = ["Razieh Behjati <razieh@google.com>"]
 edition = "2021"
 license = "Apache-2.0"
 
+[dependencies]
+bincode = "*"
```

Comment:
Is the use of bincode and prost here only temporary untill everything is converted to flatbuffers, or do we plan on having them around longer term?